torch/_tensor_docs.py

```diff
@@ -2384,6 +2384,14 @@ def callable(a, b) -> number
     0.012766935862600803
 """)
 
+add_docstr_all('to_sparse',
+               r""""
+to_sparse() -> Tensor
+
+Returns a sparse copy of the tensor.  Torch supports sparse tensors
+in coordinate format. See https://pytorch.org/docs/stable/sparse.html.
```

Comment:
Please use sphinx tag reference so that different versions will point to the correct page.